File: src/main/java/com/tom/Hello.java
package com.tom;

public class Hello {
    public static void main(String[] args){
//        System.out.println("Hello World!!");
        Person p = new Person( "Tom",66.5f, 1.7f);
        p.hello();
        p.weight = 66.5f;
        p.height = 1.7f;
        System.out.println(p.bmi());
        int score = 88;
        System.out.println( score >80 || score<90);
        char c = 'A';
        System.out.println(c > 'a');


        /*int age;
        age = 22;
        char c = '中';
        Character c2 = 'A';
        byte b = 30;
        float weight = 65.5f;
        boolean enroll = false;
        String s = new String("ABCD");
        System.out.println(enroll);*/
    }
}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main(args: Array<String>) {
//    println("Hello World!")
//  Human().hello()
    val h = Human(weight = 66.5f,height=1.7f)
    println(h.bmi())
    val score = 88
    println(score > 60)
    val c:Char = 'A'
    println(c.toInt() > 60)
    /*h.hello()
    var age : Int = 25;
    age = age.plus(1)
    val weight = 65.5
    val enroll : Boolean = true
    println(age)*/
}

class Human(var name:String ="" ,var weight:Float,var height:Float) {
    init {
        println("test$weight")
    }
//    constructor(name:String,weight:Float,height:Float):this(weight,height)

    fun bmi():Float{
        val bmi = weight/(height*height)
        return bmi
    }

    fun hello(){
        println("Hello Kotlin")
    }

}